includes/core.php

```diff
@@ -502,6 +502,13 @@ function get_app_rate_limits_markup( $rate_limits ) {
  * @return string
  */
 function get_rate_limits_markup( $title, $remaining, $limit, $reset ) {
+	// If reset time is less than current time, then rate limit is not available.
+	if ( $reset && $reset < time() ) {
+		return sprintf(
+			'<p>%s</p>',
+			esc_html__( 'No X/Twitter rate limit available yet. Make a post to X/Twitter first.', 'autoshare-for-twitter' )
```

Comment:
As far as the message here goes, do we want to still refer to things as `X/Twitter` or should we start just using `X`? Not sure how we refer to things in other places right now

Reply:
We are still using `X/Twitter` in most places, but I think `X` is now well-known enough that we can drop `Twitter` from all places. @jeffpaul, what do you think?

Reply:
For what it's worth, my opinion here is for any new strings we introduce, we should just use `X` and can look to update existing strings in a separate PR. I'm also fine though if we want to continue to use `X/Twitter`, just noting that reads a little awkwardly and the change has been in place long enough that I think anyone using this plugin would understand what just `X` means

Reply:
Probably fine now to update to just "Autopost for X" and "X" throughout.